### ISSUE TITLE
singularity inspect --app with --runscript and --helpfile fatal

### ISSUE THREAD

**Issue by jscook2345** [apptainer/singularity#3059]
## Version of Singularity:

3.0.1

Using `singularity/examples/apps/Singularity.cowsay`.

There is a `%apphelp lolcat` so I would expect this to work.
<pre>
[justin@ip-172-31-94-96 inspect]$ singularity inspect --app lolcat -H cowsay.sif
FATAL:   exit status 126
</pre>

There is a `%apprun lolcat` so I would expect this to work.
<pre>
[justin@ip-172-31-94-96 inspect]$ singularity inspect --app lolcat -r cowsay.sif
FATAL:   exit status 127
</pre>

**Comment by jscook2345** [comment]
Additionally, there is no `test` section, but this fatals instead of giving some sort of error.

<pre>
[justin@ip-172-31-94-96 inspect]$ singularity inspect --app lolcat -t cowsay.sif
FATAL:   exit status 127
</pre>

Possibly related?

**Comment by phpHavok** [comment]
This appears to be fixed in `3.2.0-rc1`.

```
[jacob@gideon temp]$ singularity inspect --app lolcat -H cowsay.sif
==helpfile==
    lolcat is the best app


[jacob@gideon temp]$ singularity inspect --app lolcat -r cowsay.sif
==runscript==
#!/bin/sh

    lolcat "$@"


[jacob@gideon temp]$ singularity inspect --app lolcat -t cowsay.sif
```

**Comment by WestleyK** [comment]
Closing, confirmed this is fixed.
